src/controllers/applicationsContentController.js

```diff
@@ -0,0 +1,304 @@
+    const mockThrottlingPoliciesMetaDataPath = path.join(process.cwd(), filePrefix + '../mock/Applications', 'throttlingPolicies.json');
+    const mockThrottlingPoliciesMetaData = JSON.parse(fs.readFileSync(mockThrottlingPoliciesMetaDataPath, 'utf-8'));
+    return mockThrottlingPoliciesMetaData.list;
+}
+
+async function getAPIMThrottlingPolicies() {
+    try {
+        const response = await axios({
+            method: 'GET',
+            url: controlPlaneUrl + '/throttling-policies/application',
+            headers: {
+                Authorization: `Bearer ${token}`,
+            },
+            httpsAgent
+        });
+        return response.data.list;
+    } catch (error) {
+        console.error('Error fetching throttling policies:', error.message);
+    }
+}
+
+// ***** Load Application *****
+
+const loadApplication = async (req, res) => {
+    const orgName = req.params.orgName;
+    const applicationId = req.params.applicationid;
+    let html, templateContent, metaData;
+    if (config.mode === constants.DEV_MODE) {
+        metaData = await getMockApplication();
+        templateContent = {
+            applicationMetadata: metaData,
+            baseUrl: constants.BASE_URL + config.port
+        }
+    } else {
+        metaData = await getAPIMApplication(applicationId);
+        templateContent = {
+            applicationMetadata: metaData,
+            baseUrl: '/' + orgName
+        }
+    }
+    html = renderTemplate('../pages/application/page.hbs', filePrefix + 'layout/main.hbs', templateContent, true);
+    res.send(html);
+}
+
+const loadApplicationForEdit = async (req, res) => {
+    const orgName = req.params.orgName;
+    const applicationId = req.params.applicationid;
+    let html, templateContent, metaData;
+    if (config.mode === constants.DEV_MODE) {
+        metaData = await getMockApplication();
+        throttlingMetaData = await getMockThrottlingPolicies();
+        templateContent = {
+            applicationMetadata: metaData,
+            throttlingPoliciesMetadata: throttlingMetaData,
+            baseUrl: constants.BASE_URL + config.port
+        }
+    } else {
+        metaData = await getAPIMApplication(applicationId);
+        throttlingMetaData = await getAPIMThrottlingPolicies();
+        templateContent = {
+            applicationMetadata: metaData,
+            throttlingPoliciesMetadata: throttlingMetaData,
+            baseUrl: '/' + orgName
+        }
+    }
+    html = renderTemplate('../pages/edit-application/page.hbs', filePrefix + 'layout/main.hbs', templateContent, true);
+    res.send(html);
+}
+
+async function getMockApplication() {
+    const mockApplicationMetaDataPath = path.join(process.cwd(), filePrefix + '../mock/Applications/DefaultApplication', 'DefaultApplication.json');
+    const mockApplicationMetaData = JSON.parse(fs.readFileSync(mockApplicationMetaDataPath, 'utf-8'));
+    return mockApplicationMetaData;
+}
+
+async function getAPIMApplication(applicationId) {
```

Comment:
We can utilize the common API invocation call for this purpose

Reply:
Will not that from next PR after all the codes are merged so a common one for all can be used.